modules/oe_migration_workflow/tests/src/Unit/Plugin/migrate/process/SetWorkflowStateTest.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_migration_workflow\Unit\Plugin\migrate\process;
+
+use Drupal\Core\Config\ConfigManagerInterface;
+use Drupal\Core\Config\Entity\ConfigEntityTypeInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState;
+use Drupal\migrate\Row;
+use Drupal\Tests\migrate\Unit\process\MigrateProcessTestCase;
+use Drupal\workflows\WorkflowInterface;
+use stdClass;
+
+/**
+ * @coversDefaultClass \Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState
+ *
+ * @group oe_migration
+ */
+class SetWorkflowStateTest extends MigrateProcessTestCase {
+  /**
+   * The ID of the plugin under test.
+   *
+   * @var string
+   */
+  protected $pluginId = 'oe_migration_workflow_set_workflow_state';
```

Comment:
Mmm, is this the plugin ID? Isn't it `oe_migration_set_workflow_state `?

Also, this is not reused. You don't need the variable here, you can directly use the value inside `initializePlugin`

Reply:
Yes, I  usually declare it to have more or less the code of the same way, but in this case we can remove it.